### ISSUE TITLE
Mesh for a complicated cross-section of a composite slab

### ISSUE THREAD

**Issue by nelly-22** [gmsafir/gmsafir#20]
Hello,
I'm Nelly and I'm a student of the Leibniz University of Hannover. I'm using Safir for my final thesis. 

I created a cross-section of a composite slab of concrete, steel and reinforcements for a 2D Thermal Analysis and I have some problems with the mesh. I tried to make many sections to have a regular mesh. When I calculate the IN.File, I receive an error for one element (see picture). I checked the surfaces and the order/direction of the lines too, but I can't find the solution. Can you help me? Do you have tips to do a good mesh for this kind of cross-section?

Best Regards
Nelly  

![grafik](https://user-images.githubusercontent.com/108728919/179744215-8988907d-b847-4cd0-b8c9-0c9296f48338.png)
![grafik](https://user-images.githubusercontent.com/108728919/179744497-a384bb42-29cf-42c4-814c-8e1da35c21f5.png)

**Comment by nelly-22** [comment]
[Slab.geo.txt](https://github.com/gmsafir/gmsafir/files/9140846/Slab.geo.txt)
[Slab.g4s.txt](https://github.com/gmsafir/gmsafir/files/9140847/Slab.g4s.txt)

**Comment by gmsafir** [comment]
Please, correct first the geometry problems mentioned in the log window (in red characters, down right), these might contribute to cause your problem
![image](https://user-images.githubusercontent.com/99477211/179924791-06f3c69f-7a06-45d2-9ea1-f40eef786f72.JPG)

**Comment by nelly-22** [comment]
Thanks for you answer and your tip!

Probably I had an error in the script with the order of the numbers. I think the command "Boolean Difference" was the problem. In the beginning I had two surfaces in the middle - "concrete" and the steel profile. So I used the Boolean Command to cut out the steel profil of the concrete section. But then the mesh didn't work. So I did many surfaces and connected them with the command "physikcal groups". Then it worked well. Maybe that was the fault.
